### DISCUSSION TITLE
render.nvim

### DISCUSSION THREAD

**Discussion by roland-5** [nvim-neorg/neorg#791] [General]
Hi,

is this project could somehow help Neorg? https://github.com/SeniorMars/render.nvim

**Comment by max397574** [comment]
probably
but I'm not exactly sure how well it could help
this guy is talking about the completely independent things (images and anti-conceal)
for images there is hologram.nvim which vhyrro contributes to these days
anti-conceal is something that needs to be done in core

**Comment by pysan3** [comment]
Checkout https://github.com/nvim-neorg/neorg/issues/971

**Comment by Atreyu-94** [comment]
I was just thinking that Typst's ease of getting svg and PDF in addition to the syntax of the equations could be very helpful.

For example, entering the markup of Typst equations in Unicode will render them correctly in Typst. Which is very helpful when you are reviewing the source code of the equation, which is extremely clean and readable.

**Reply by pysan3** [reply]
I'm not that familiar with Typst but is this related?

https://github.com/nvim-neorg/neorg/pull/1133

**Reply by Atreyu-94** [reply]
Indeed. LaTeX has been the standard in mathematical writing for decades and has never had a rival.

However, since [Typst](https://typst.app/) emerged, mathematical writing is extremely ergonomic. To the point that it makes me think that this time it is a reliable substitute for LaTeX.

Writing mathematics in plain text with it is practically using unicode symbols that we can achieve with the plugin: [kdheepak/cmp-latex-symbols](https://github.com/kdheepak/cmp-latex-symbols).

In this way we obtain formulas perfectly rendered to PDF and SVG, in addition to being extremely readable in plain text. LaTeX ones can be very cumbersome in plain text when many operators are used.

**Comment by avonmoll** [comment]
I definitely think that neorg should support Typst (as an alternative to LaTeX) for math mode. Typst is exploding in popularity due to its ease of use, speed, and extensibility. The render speed is insane. Plus, as a dependency, it's much lighter-weight than LaTeX.
